astropy/units/quantity.py

```diff
@@ -1423,6 +1427,13 @@ def to_string(self, unit=None, precision=None, format=None, subfmt=None):
             - 'latex_inline': Return a LaTeX-formatted string that uses
               negative exponents instead of fractions
 
+        formatter : str, callable, optional
+            The formatter to use for the value. If a string, it should be a
+            valid format specifier using Python's mini-language. If a callable,
+            it will be treated as the default formatter for all values and will
+            overwrite default Latex formatting for exponential notation and complex
+            numbers. If not provided, the default formatter will be used.
```

Comment:
add "numbers. If a `dict`, it will be passed directly to `~numpy.array2string`. If not..."